CODEOWNERS

```diff
@@ -0,0 +1 @@
+@ankurtyagi-sdk @pratap-akhand @varunshrivastava2502
```

Comment:
suggestion: move to a `.github` directory?

Reply:
@ankursarin we have moved the .docx document to markdown (.md) file. All images inside the user documentation have been linked with .md file and the images are stored in "images" folder on repository base location.

Also moved CODEOWNERS file to .github directory.

Reply:
sounds good